README.md

```diff
@@ -34,7 +34,8 @@
 
 # 📖 To-do-list <a name="about-project"></a>
```

Comment:
- [OPTIONAL ] Always make your README file as descriptive as possible. I would suggest that you add detailed instructions on how to run test in the Run test section in this format. 
- Install Jest as a development dependency for npm `npm install --save-dev jest` and 
- Run `npm run test` in the command line